Lib/test/test_enum.py

```diff
@@ -4449,6 +4449,12 @@ def test__all__(self):
 COMPLEX_A = 2j
 COMPLEX_B = 3j
 
+class _ModuleWrapper:
```

Comment:
I like this.  Please switch over the other `_convert_` tests to use it, which will allow the removal of the `setUp()` methods in those tests.